src/pages/videolist/Videolist.jsx

```diff
@@ -41,31 +44,55 @@ export function Videolist() {
             filteredVideos?.map(({ _id, title, videoId, thumnailHigh }) => {
               return (
                 <li key={_id} className="card children-stacked card-dismiss">
-                  <div className="card-media">
-                    <img src={thumnailHigh.url} alt="card-img" />
-                  </div>
-                  <div className="card-head d-flex">
-                    <div className="card-header children-stacked">
-                      <div className="card-title">
-                        {"title title title title title title title title"}
-                      </div>
-                      <div className="card-author gray-text">
-                        by Aishwarya B
-                      </div>
+                  <Link to={`/explore/video/${_id}`}>
+                    <div className="card-media">
+                      <img src={thumnailHigh.url} alt="card-img" />
                     </div>
+                  </Link>
 
-                    <button className="btn btn-link icon-more">
+                  <div className="card-head d-flex p-rel">
+                    <Link to={`/explore/video/${_id}`}>
+                      <div className="card-header children-stacked">
+                        <div className="card-title">
+                          {"title title title title title title title title"}
```

Comment:
I'm assuming you're going to get the title here, replace it with the video title here

Reply:
Hey yes this is until i implement the features, will change in upcoming pushes